nixos/modules/system/boot/systemd.nix

```diff
@@ -587,6 +587,14 @@ in
       '';
     };
 
+    services.journald.forwardToSyslog = mkOption {
+      default = (if config.services.rsyslogd.enable || config.services.syslog-ng.enable then true else false);
```

Comment:
Please drop the if statement, `config.services.rsyslogd.enable || config.services.syslog-ng.enable` is enough here.

Also, please set a `defaultText` to the string of the expression of `default` so the manual won't have to be rebuilt if this value changes.